api/v1alpha1/kubeadmbootstrapconfig_types_test.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
```

Comment:
Any reason why we aren't using the gloang `"testing"` package instead?

Reply:
Throughout

Reply:
+2 for `gomega`,  golang `testing` is fine for some basic tests, but once you start doing extensive unit and integration testing that this controller will need the lack of matchers really becomes painful 

+0 for `ginkgo` - I think BDD is an individual style choice, and whether to use it or not probably amounts to how many potential contributors prefer that style.  I personally don't prefer BDD, but I don't dislike it enough to give it a -1

Reply:
This is generated by the kubebuilder scaffolding.

Reply:
This is a generated file and also the beginning of scaffolding. Gonna leave this.